src/service/router/test_router.py

```
@@ -0,0 +1,106 @@
+with (
+    mock.patch.dict(os.environ, {'OSMO_POSTGRES_PASSWORD': 'test-password'}),
+    mock.patch('fastapi.applications.FastAPI.add_middleware'),
+):
+    router = importlib.import_module('src.service.router.router')
+
+
+class _FakeRequest:
+    cookies = {
+        '_osmo_router_affinity': 'sticky-session',
+        'ignored': 'cookie',
+    }
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)